testing/matrix-sdk-test/src/test_json/keys_query_sets.rs

```diff
@@ -919,9 +981,30 @@ impl PreviouslyVerifiedTestData {
         ruma_response_from_json(&data)
     }
 
-    pub fn device_1_keys_payload_carol() -> Value {
+    /// Device ID of Carol's signed device.
+    ///
+    /// The device is returned as part of
+    /// [`Self::carol_keys_query_response_signed`] and
+    /// [`Self::carol_keys_query_response_unsigned`].
+    pub fn carol_signed_device_id() -> &'static DeviceId {
```

Comment:
Seeing as the users have 2 devices each, would it be better to always call them `_signed`/`_unsigned` and drop the `_1` numbering scheme used above (`Self::bob_device_1_id`)?

Reply:
The problem is that which of bob's devices are signed depends on whether you use `bob_keys_query_response_signed` or `bob_keys_query_response_rotated`.